cookbook/security/api_key_authentication.rst

```diff
@@ -318,6 +318,10 @@ using the ``simple_preauth`` key:
                     simple_preauth:
                         authenticator: apikey_authenticator
 
+            providers:
+                simple_preauth:
+                    id: your_api_key_user_provider
+
```

Comment:
@weaverryan Updated as per your comments, I think adding the service makes sense. I actually had this before, but then removed it again due to the reference. But that should not be an issue to maintain.

I also changed everything to your favorite, the AppBundle.. :)